packages/create-bison-app/template/tests/e2e/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../tsconfig.json",
+  "extends": "../../cypress/tsconfig.json",
   "compilerOptions": {
     // "baseUrl": "../node_modules",
```

Comment:
I know this isn't changed, is there a reason this is here & commented out? Should we note why it's there? I know in one of our apps I'm using that line so curious.

Reply:
As far as I could tell from the commit history, I don't think that line was ever significant so I've removed it.